src/pages/MyInformation/hooks/useGetMyDesigns.ts

```diff
@@ -11,17 +11,37 @@ const getMyDesigns = async (pathname: string): Promise<VendorQueryResult> => {
   const { data } = await request({
     pathname,
     method: 'get',
-    params: {
-      count: DEFAULT_LIST_LENGTH,
-    },
     accessToken: getAccessToken(),
   });
 
   return data;
 };
 
-export const useGetMyDesigns = (): SWRResponse<VendorQueryResult, unknown> => {
-  const response = useSWR('designs/my', (pathname) => getMyDesigns(pathname));
+export const useGetMyDesigns = (): SWRInfiniteResponse<
+  VendorQueryResult,
+  ListResponse<DesignItemResponse>
+> => {
+  function getKey(
+    pageIndex: number,
+    previousPageData: VendorQueryResult | null,
+  ) {
+    const isLastCursor = previousPageData && !previousPageData.payload;
+
+    if (isLastCursor) {
+      return null;
+    }
+
+    const isFirstCursor = pageIndex === 0;
+    const lastCursor = previousPageData?.meta?.last_cursor;
+    const afterValue =
+      isFirstCursor || lastCursor == null ? `` : `&after=${lastCursor}`;
```

Comment:
```suggestion
    const afterValue =
      isFirstCursor || lastCursor == null ? '' : `&after=${lastCursor}`;
```
p4: 여기 한 문장에 분기문 두개라 조금 헷갈리네요 😢 
함께 표현할 필요 없다면 isFirstCursor에 대한 분기는 별도로 분리하면 좀 더 목적을 이해하기 편할 것 같아요.

Reply:
좋습니다!